src/Sarif.UnitTests/SarifTransformerTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using FluentAssertions;
```

Comment:
Copyright notice on new files (see existing files for example).

Reply:
Fixed